test/docker/fs/etc/colonus/anax-combined.config.tmpl

```diff
@@ -10,6 +10,7 @@
         "PublicKeyPath": "/root/.colonus/mtn-publicKey.pem",
         "DefaultServiceRegistrationRAM": 1024,
         "ExchangeURL": "${EXCH_APP_HOST}/",
+        "CACertsPath": "/certs/css.crt",
```

Comment:
There is already a CACertsPath field in this file...and why arent you setting it in the remote config?

Reply:
Sorry the naming convention is wrong. I had originally named it remote due to the fact that it was being used for DEV. I need to change it from Remote to no-cert, because now the remote is used for anytime that does not require a cert and regular is for when it does require a cert. I had to include the CACertsPath, because I had a command that was failing even when using HZN_SSL_SKIP_VERIFY=1.